docs/conf.py

```diff
@@ -181,5 +181,94 @@
      [author], 1)
 ]
 
+
```

Comment:
We should move this python code into a sphinx extension module.

We can then add the extension to the `requirements.txt` and the extension list found at https://github.com/google/gf180mcu-pdk/blob/main/docs/conf.py#L51-L62

We can probably store the extension in the repository at https://github.com/google/open-source-pdks